tsdb/engine/tsm1/mmap_windows.go

```diff
@@ -0,0 +1,11 @@
+// portability wrapper for Mmap/Munmap
+// +build windows
+
+package tsm1
+
+import (
+	"golang.org/x/sys/windows"
+)
+
+var Mmap = windows.Mmap
```

Comment:
No, it doesn't. I verified this on a 32-bit Windows VM (I don't have access to 64-bit Windows). Sorry for the confusion.